interface/themes/style_prism.css

```diff
@@ -0,0 +1,1507 @@
+/* Theme: Prism
+ *
+ * @package LibreHealth EHR
+ * @author MigDinny <github.com/MigDinny>
+ * @link http://librehealth.io
+ * 
+ * This style is a replacement and its code (not its design) is based on Tan Theme
+ * This Theme must have some color sets associated (Green, Blue and Orange) to be chosen
+ */
+
+/* Imports area */
+
+/* How to use: font-family: 'Roboto', sans-serif; */
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```

Comment:
This is a medical application (outside calls not good), and may be air-gapped.  You need to add the resource to /assets/fonts.

Reply:
Had to add the .ttf on theme_assets because there was no other way to do it. Tried assets/fonts but on CSS I can't access it because php is not available there to call the globals.